README.md

```diff
@@ -48,18 +48,18 @@ $ kubectl apply -f deploy/upstream
 ```
 
 #### Installing an operator using Marketplace
-The following section assumes that Marketplace was installed in the `openshift-marketplace` namespace. For Marketplace to function you need to have at least one `OperatorSource` CR present on the cluster. To get started you can use the [community](deploy/examples/operatorsource.cr.yaml) OperatorSource. If you are on an OKD cluster, you can skip this step as it is installed by default.
+The following section assumes that Marketplace was installed in the `openshift-marketplace` namespace. For Marketplace to function you need to have at least one `OperatorSource` CR present on the cluster. To get started you can use the [upstream-community-operators](deploy/examples/operatorsource.cr.yaml) OperatorSource. If you are on an OKD cluster, you can skip this step as it is installed by default.
 ```bash
 $ kubectl apply -f deploy/examples/operatorsource.cr.yaml
 ```
 Once the `OperatorSource` has been successfully deployed, you can discover the operators available using the following command:
 ```bash
-$ kubectl get opsrc community-operators -o=custom-columns=NAME:.metadata.name,PACKAGES:.status.packages -n openshift-marketplace
-NAME                  PACKAGES
-community-operators   federationv2,svcat,metering,etcd,prometheus,automationbroker,templateservicebroker,cluster-logging,jaeger,descheduler
+$ kubectl get opsrc upstream-community-operators -o=custom-columns=NAME:.metadata.name,PACKAGES:.status.packages -n openshift-marketplace
+NAME                           PACKAGES
+upstream-community-operators   jaeger
 ```
 
-Now if you want to install the `descheduler` and `jaeger` operators, create a `CatalogSourceConfig` CR as shown below:
+Now if you want to install the `jaeger` operator, create a `CatalogSourceConfig` CR as shown below:
```

Comment:
I would like to use this example to call out that you use the same `CatalogSourceConfig` to install multiple operators.

Reply:
We don't have descheduler or any other operator besides jaeger in the `upstream-community-operator` OperatorSource. We could update the README to reflect the point above as soon as we have another operator in the `upstream-community-operator` OperatorSource.

Reply:
Again, we are showing examples here. It doesn't need to reflect exactly what is present in the repository which is going to be changing. We are not going to keep updating the README each time the Quay repo changes.